docs/building.md

```diff
@@ -78,7 +78,12 @@ You can generate the known metadata, but you'll need the `stack` tool installed,
 [The Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/) tutorial for instructions on how to obtain it.
 
 ```bash
-make metadata --output=metadata.json
+make metadata \
+    frozen_token_symbol=frozen_token \
+    frozen_token_name="Frozen Token" \
```

Comment:
May be it is better to write down "BaseDAO Frozen Token" here, as a token name of "Frozen Token" might appear wierd without context, when it is displayed in a wallet or some other device

Reply:
Updated this.